__tests__/app.test.js

```diff
@@ -58,4 +58,51 @@ describe("GET /api", () => {
         ).toBe(true);
       });
   });
+});
+
+describe("GET /api/articles/:article_id", () => {
+  test("GET:200 sends an article by id", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then((res) => {
+        expect(res.body.article).toMatchObject({
+          article_id: 1,
+          title: "Living in the shadow of a great man",
+          topic: "mitch",
+          author: "butter_bridge",
+          body: "I find this existence challenging",
+          created_at: "2020-07-09T20:11:00.000Z",
+          votes: 100,
+          article_img_url:
+            "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+        });
+        expect(res.body.article).toMatchObject({
+          article_id: expect.any(Number),
```

Comment:
You probably only need one of these. Ideally, you'd want to check the article_id is the one you're looking for, not just any number. You've already checked it above, so not an issue, but if you were just doing .toMatchObject, you'd need to make sure you check the article_id is accurate